src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -140,6 +146,30 @@ class FormPage extends React.Component {
       }
     }
 
+    // Bypass the SchemaForm and render the custom component
+    // NOTE: I don't think FormPage is rendered on the review page, so I believe
+    // onReviewPage will always be false here
+    if (typeof route.pageConfig.CustomPage === 'function') {
+      return (
+        <div className={pageClasses}>
```

Comment:
Does the wrapping div do anything for us here? Could the classes be applied directly to `CustomPage`?

Reply:
It doesn't get us _much,_ no. I don't know what happens if all pages aren't wrapped in a `form-panel` class, so I opted to keep it because it didn't seem like it'd hurt anything. :man_shrugging: